src/lib.rs

```diff
@@ -4834,6 +4864,14 @@ pub trait IntoByteSliceMut<'a>: ByteSliceMut + Into<&'a mut [u8]> {}
 #[allow(clippy::undocumented_unsafe_blocks)]
 unsafe impl<'a> ByteSlice for &'a [u8] {}
 
+// TODO(#429): Add a "SAFETY" comment and remove this `allow`.
+#[allow(clippy::undocumented_unsafe_blocks)]
+unsafe impl<'a> CopyableByteSlice for &'a [u8] {}
+
+// TODO(#429): Add a "SAFETY" comment and remove this `allow`.
+#[allow(clippy::undocumented_unsafe_blocks)]
+unsafe impl<'a> CloneableByteSlice for &'a [u8] {}
```

Comment:
It might be nice to burn down these TODOs while we're here, rather than add to our safety TODO backlog, but we can also punt on this.

Reply:
My reasoning is that we've just split the trait's safety obligations into multiple traits, and not added any new obligations. Happy to do this now if you want - I just figured it'd be good not to block on that.